util/msys2_install.sh

```diff
@@ -8,25 +8,27 @@ installflip=false
 util_dir=$(dirname "$0")
 
 echo "Installing dependencies needed for the installation (quazip)"
-pacman --needed -S base-devel mingw-w64-x86_64-toolchain mingw-w64-x86_64-clang msys/git msys/p7zip msys/python3 msys/unzip
+pacman --needed --noconfirm --disable-download-timeout -Sy base-devel mingw-w64-x86_64-toolchain mingw-w64-x86_64-clang msys/git msys/p7zip mingw-w64-x86_64-python3-pip msys/unzip
 
 source "$dir/win_shared_install.sh"
 
 function install_avr {
     rm -f -r "$avrtools"
-    wget "http://ww1.microchip.com/downloads/en/DeviceDoc/avr8-gnu-toolchain-3.6.1.1752-win32.any.x86.zip"
+    wget "https://blog.zakkemble.net/download/avr-gcc-8.3.0-x86-mingw.zip"
```

Comment:
Was about to just merge this, but I noticed this is the 32-bit version. Do we want to be using 64-bit instead?

Reply:
Ideally, yes, I think we should be using the 64 bit binary, but even still, we can't be 100% certain that people are using 64 bit OS' (uuugh).  

If we have a good way to test this ... then that would be ideal, but I'm not sure how.  So for now, I think defaulting to the x86 version may be safest.